package.json

```diff
@@ -88,5 +94,25 @@
   },
   "lint-staged": {
     "*.{js,css,md}": "prettier --write"
+  },
+  "release": {
+    "branches": [
+      "main",
+      "next"
+    ],
+    "plugins": [
+      [
+        "@semantic-release/commit-analyzer",
+        {
+          "preset": "conventionalcommits"
+        }
+      ],
+      "@semantic-release/release-notes-generator",
```

Comment:
I need to check on [test semantic release](https://github.com/matthieuh/test-semantic-release) if the `preset` is needed for release notes generator